scripts/verify-structure.py

```
@@ -629,31 +851,104 @@ def check_marketplace_structure() -> dict[str, Any]:
-        plugin_results = check_plugin_manifest(plugin_dir)
+        plugin_results = check_plugin_manifest(
+            plugin_dir,
+            marketplace_entry=plugin_entry,
+            strict_mode=strict_mode
+        )
         result['plugin_results'][plugin_name] = plugin_results
 
     return result
 
 
+def calculate_exit_code(result: dict[str, Any], strict: bool = False) -> int:
```

Ruff (0.14.4)

Boolean-typed positional argument in function definition

(FBT001)